src/components.js

```diff
@@ -0,0 +1,41 @@
+const text = (elem, text, className) => {
+  let t = document.createElement(`${elem}`);
+  t.textContent = `${text}`;
+  t.classList.add(`${className}`);
+  return t;
+};
+
+function tabs() {
+  const navbar = text("ul", "", "navbar");
+  for (var i = 0; i < arguments.length; i += 1) {
+    let el = text("li", `${arguments[i]}`, "nav-item");
+    navbar.appendChild(el);
+  }
+  return navbar;
+}
+
+function menuItems(arr) {
+  const menu = text("section", "", "menu");
+  for (var i = 0; i < arr.length; i += 1) {
```

Comment:
**REQUIRED**
* Let's make use of es6 syntax. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/es6_requirement.png)).